composer.json

```diff
@@ -20,7 +23,7 @@
         "symfony/config": "^3.4 || ^4.0",
         "symfony/dependency-injection": "^3.4 || ^4.0",
         "symfony/framework-bundle": "^3.4 || ^4.0",
-        "symfony/http-foundation": "^3.4 || ^4.0",
+        "symfony/http-foundation": "^3.4.13 || ^4.0",
```

Comment:
The downside of `--prefer-lowest` (the problem applies to `4.0.0` and `4.1.0` too).